logprep/generator/http/controller.py

```diff
@@ -32,7 +30,11 @@ def __init__(self, **kwargs):
         self._setup_logging()
         self.thread_count: int = kwargs.get("thread_count")
         self.input: Input = Input(self.config)
-        self.output = create_output(kwargs)
+        self.output = self._create_output(kwargs)
+
+    def _create_output(self, kwargs):
```

Comment:
if subclasses have to implement something make the method abstract by decorate them with `@abstractmethod` (have a look at logprep.abc)